### PR TITLE
Fix/start staking time formatters

### PR DESCRIPTION
**Pull request by antonijzelinskij**
#85ztyp5pc

### CODE REVIEW

**Comment by stepanLav** [comment]
Task linked: [CU-85ztyp5pc Android. Fix time formaters on start staking](https://app.clickup.com/t/85ztyp5pc)

File: common/src/test/java/io/novafoundation/nova/common/utils/formatting/SpannableFormatterTest.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package io.novafoundation.nova.common.utils.formatting
+
+class SpannableFormatterTest {
+}
```
Comment (valentunn) [inline common/src/test/java/io/novafoundation/nova/common/utils/formatting/SpannableFormatterTest.kt:4]: Have you planned to write some tests here?

**Review by valentunn — APPROVED** [review]
Great work!

File: common/src/test/java/io/novafoundation/nova/common/utils/formatting/SpannableFormatterTest.kt (hunk 1 above)
Comment (antonijzelinskij) [reply common/src/test/java/io/novafoundation/nova/common/utils/formatting/SpannableFormatterTest.kt]: No

**Pull request merged**